addons/l10n_fr_pos_cert/models/account_bank_statement.py

```diff
@@ -27,5 +27,5 @@ def _unlink_except_created_by_pos(self):
             statement = st_line.statement_id
             if not statement.company_id._is_accounting_unalterable() or not statement.journal_id.pos_payment_method_ids:
                 continue
-            if statement.state != 'open':
+            if st_line.state != 'draft':
```

Comment:
this is redundant with the existing check in point_of_sale I think
```py
    pos_session_id = fields.Many2one('pos.session', string="Session", copy=False)

    @api.ondelete(at_uninstall=False)
    def _unlink_except_linked_to_pos_session(self):
        for bsl in self:
            if bsl.pos_session_id:
                raise UserError(_("You cannot delete a bank statement line linked to Point of Sale session."))

```
This check can be removed imo